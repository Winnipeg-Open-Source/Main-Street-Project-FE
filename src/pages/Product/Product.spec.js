import React from 'react';
import renderWithThemeAndRouter from 'tests/helpers/renderWithThemeAndRouter';
import ProductPage from './Product';

describe ('ProductPage', () => {
    it ('renders without crashing', () => {
        const { asFragment } = renderWithThemeAndRouter(<ProductPage />);

        expect(asFragment()).toMatchSnapshot();
    });
});
