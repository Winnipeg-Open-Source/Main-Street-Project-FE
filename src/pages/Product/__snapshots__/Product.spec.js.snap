// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductPage renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-pbJYR hNzbLI sc-pkfPC jpeBiC"
  >
    <div>
      Product
    </div>
  </div>
</DocumentFragment>
`;
